SYCL/Regression/host_unified_memory.cpp

```diff
@@ -0,0 +1,48 @@
+// REQUIRES: gpu && linux
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER SYCL_HOST_UNIFIED_MEMORY=1 %t.out
+
+#include <CL/sycl.hpp>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+
+using namespace cl::sycl;
+
+static buffer<char, 1> *inBufP = nullptr;
+
+int main(int argc, char *argv[]) {
+  queue Q;
+  auto BE =
+      (bool)(Q.get_device()
+                 .template get_info<sycl::info::device::opencl_c_version>()
+                 .empty())
+          ? "L0"
+          : "OpenCL";
+  device dev = Q.get_device();
+  size_t max_compute_units = dev.get_info<info::device::max_compute_units>();
```

Comment:
Minor.
Information about compute units seems to be irrelevant to the test case.